protos/flow.proto

```diff
@@ -631,3 +631,8 @@ message GetDefaultPartitionKeyForTablesInput {
 message GetDefaultPartitionKeyForTablesOutput {
   map<string, string> table_default_partition_key_mapping = 1;
 }
+
+message GetFlowConfigAndWorkflowIdFromCatalogOutput {
+  FlowConnectionConfigsCore flow_connection_configs = 1;
+  string original_workflow_id = 2;
```

Comment:
```suggestion
  string workflow_id = 2;
```
We aren't ever changing the workflow id so distinguishing the original would be confusing